.gitignore

```diff
@@ -1,21 +1,21 @@
-# logs
-*.log
-
-# temporary
-*.swp
-
-# filesystem
-.DS_Store
-.DS_Store?
-._*
-.Spotlight-V100
-.Trashes
-ehthumbs.db
-Thumbs.db
-*~
-
-# packages
-node_modules
-
-# builds
-opt/test/build.js
+# logs

```

Comment:
No idea! I'm guessing it's the newline types. I'll check out an old version of this file, hopefully that fixes it.